frontend/app/views/joiner/form/payment.scala.html

```diff
@@ -1,66 +1,75 @@
 @import com.gu.i18n.CountryGroup
-@import com.gu.memsub.promo.{NewUsers, ValidPromotion}
 @import com.gu.memsub.subsv2.CatalogPlan.PaidMember
 @import com.gu.memsub.subsv2.MonthYearPlans
-@import com.gu.salesforce.Tier.Supporter
-@import tracking.AppnexusPixel
+@import com.gu.salesforce.Tier.{Supporter, Partner}
 @import views.html.helper._
 @import views.support.DisplayText._
 @import views.support.MembershipCompat._
 @import views.support.{CountryWithCurrency, IdentityUser, PageInfo, PaidPlans}
+@import tracking.AppnexusPixel
+
+@import abtests.PayPalTestVariants
 @(plans: MonthYearPlans[PaidMember],
     countriesWithCurrencies: List[CountryWithCurrency],
     idUser: IdentityUser,
     pageInfo: PageInfo,
-    validPromo: Option[ValidPromotion[NewUsers]],
     countryGroup: Option[CountryGroup],
-    touchpointBackendResolution: services.TouchpointBackend.Resolution
+    touchpointBackendResolution: services.TouchpointBackend.Resolution,
+    paypalTestVariant: Option[String]
     )(implicit token: play.filters.csrf.CSRF.Token, request: actions.AuthRequest[_])
 
-@main(plans.tier.cta, pageInfo=pageInfo, simpleHeader=true, touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
+@main(plans.tier.cta, pageInfo=pageInfo, simpleHeader=true, margins=false, touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
 
-    <main class="page-content skin-checkout-b">
+    <main class="page-content skin-checkout-b skin-checkout-paypal">
```

Comment:
Both these skins should really get merged down into the form sass files. However, these form files are still being used on the upgrade checkouts. Since we're planning to update these checkouts to the new design, this would probably best be done at the same time.